Traceback (most recent call last):
  File "C:\Users\Charlie\.conda\envs\pythongis\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\Charlie\.conda\envs\pythongis\lib\site-packages\nbclient\client.py", line 1265, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\Charlie\.conda\envs\pythongis\lib\site-packages\jupyter_core\utils\__init__.py", line 160, in wrapped
    return loop.run_until_complete(inner)
  File "C:\Users\Charlie\.conda\envs\pythongis\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\Users\Charlie\.conda\envs\pythongis\lib\site-packages\nbclient\client.py", line 703, in async_execute
    await self.async_execute_cell(
  File "C:\Users\Charlie\.conda\envs\pythongis\lib\site-packages\nbclient\client.py", line 1021, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\Charlie\.conda\envs\pythongis\lib\site-packages\nbclient\client.py", line 915, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Crear Poligono de BBOX
from shapely.geometry import box
bbox = (-77.052133, -12.07537, -77.048287, -12.072682)
polygon = box(*bbox)
gdfBox = gpd.GeoDataFrame({"geometry": [polygon]}, crs="EPSG:4326")

# Visualizar
m = comercios_filtro.explore(m=m)
gdfBox.explore(m=m, color='red', fill=False)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Cell [1;32mIn[40], line 8[0m
[0;32m      5[0m gdfBox [38;5;241m=[39m gpd[38;5;241m.[39mGeoDataFrame({[38;5;124m"[39m[38;5;124mgeometry[39m[38;5;124m"[39m: [polygon]}, crs[38;5;241m=[39m[38;5;124m"[39m[38;5;124mEPSG:4326[39m[38;5;124m"[39m)
[0;32m      7[0m [38;5;66;03m# Visualizar[39;00m
[1;32m----> 8[0m m [38;5;241m=[39m comercios_filtro[38;5;241m.[39mexplore(m[38;5;241m=[39m[43mm[49m)
[0;32m      9[0m gdfBox[38;5;241m.[39mexplore(m[38;5;241m=[39mm, color[38;5;241m=[39m[38;5;124m'[39m[38;5;124mred[39m[38;5;124m'[39m, fill[38;5;241m=[39m[38;5;28;01mFalse[39;00m)

[1;31mNameError[0m: name 'm' is not defined
NameError: name 'm' is not defined

